packages/synapse-react-client/src/lib/containers/LoadingScreen.tsx

```diff
@@ -17,10 +21,12 @@ const loadingScreen = (
 
 type SynapseSpinnerProps = {
   size?: number // the size in pixels. default 20
+  margin?: string // the margin, default auto
```

Comment:
Make margin a SynapseSpinner prop, so an appropriate margin can be set for the BlockingLoader. Using the default 'auto' results in a larger margin that pushes the other content to the bottom of the screen.